src/Form/EntryForm.php

```diff
@@ -78,39 +89,41 @@ public function buildForm(array $form, FormStateInterface $form_state, AccountIn
    * {@inheritdoc}
    */
   public function validateForm(array &$form, FormStateInterface $form_state) {
-    $user = $form_state->getValue('account');
-    $tfa = $this->tfaManager->getProcess($user);
-    $tfa->validateForm($form, $form_state);
+    $this->tfaValidationPlugin->validateForm($form, $form_state);
   }
 
   /**
+   * For the time being, assume there is no fallback options available.
+   * If the form is submitted and passes validation, the user should be able
+   * to log in.
+   *
    * {@inheritdoc}
    */
   public function submitForm(array &$form, FormStateInterface $form_state) {
     $user = $form_state->getValue('account');
-    $tfa = $this->tfaManager->getProcess($user);
-    if(!$tfa->submitForm($form, $form_state)) {
+    //If validation failed or fallback was requested.
+    //if($form_state->hasAnyErrors()) {
       // If fallback was triggered TFA process has been reset to new validate
       // plugin so run begin and store new context.
 
-      $fallback = $form_state->getValue('fallback');
-      if (isset($fallback) && $form_state->getValue('op') === $fallback) {
-        $tfa->begin();
-      }
-      $context = $tfa->getContext();
-      $this->tfaManager->setContext($user, $context);
+//      $fallback = $form_state->getValue('fallback');
+//      if (isset($fallback) && $form_state->getValue('op') === $fallback) {
+//        $tfa->begin();
+//      }
+      //$context = $tfa->getContext();
+      //$this->tfaManager->setContext($user, $context);
       //$form_state['rebuild'] = TRUE;
-    }
-    else {
-      // TFA process is complete so finalize and authenticate user.
-      //$context = $this->tfaManager->getContext($user);
-      $tfa->finalize();
-      user_login_finalize($user);
-
-      // Set redirect based on query parameters, existing $form_state or context.
-      //$form_state['redirect'] = _tfa_form_get_destination($context, $form_state, $user);
-      $form_state->setRedirect('<front>');
-    }
+    //}
+    //else {
+    // TFA process is complete so finalize and authenticate user.
+    //$context = $this->tfaManager->getContext($user);
+    //$tfa->finalize();
```

Comment:
They are going to be replaced as part of the ongoing work. Don't want to remove it until I get the other plugin types working.